skymp5-server/cpp/server_guest_lib/MpActor.cpp

```diff
@@ -365,6 +365,40 @@ void MpActor::MpApiDeath(MpActor* killer)
   }
 }
 
+void MpActor::ModifyActorValue(espm::ActorValue av, float value)
+{
+  if ((av == espm::ActorValue::Health || av == espm::ActorValue::Stamina ||
+       av == espm::ActorValue::Magicka) == false) {
+    return;
+  }
+  BaseActorValues base =
+    GetBaseActorValues(GetParent(), GetBaseId(), GetRaceId());
+  MpChangeForm form = GetChangeForm();
+  float hp = form.healthPercentage;
```

Comment:
not sure these vars are required. can just reuse `form.healthPercentage`, etc instead